feature-libs/product-configurator/rulebased/components/attribute/types/numeric-input-field/configurator-attribute-numeric-input-field.component.ts

```diff
@@ -117,29 +96,12 @@ export class ConfiguratorAttributeNumericInputFieldComponent
       .subscribe(() => this.onChange());
   }
 
-  /**
-   * Hit when user input was changed
-   */
   onChange(): void {
-    const event: ConfigFormUpdateEvent = this.createEventFromInput();
-
-    if (!this.attributeInputForm.invalid) {
-      this.inputChange.emit(event);
-    }
+    super.onChange();
   }
 
   ngOnDestroy() {
-    this.sub.unsubscribe();
-  }
-
-  protected createEventFromInput(): ConfigFormUpdateEvent {
-    return {
-      ownerKey: this.ownerKey,
-      changedAttribute: {
-        ...this.attribute,
-        userInput: this.attributeInputForm.value,
-      },
-    };
+    super.ngOnDestroy();
```

Comment:
The same for `ngOnDestroy()`

Reply:
Let's leave it there. I am not sure whether Lifecycle methods are inherited or not.